terraform/aws_cloudflare/hosts.tpl

```diff
@@ -5,6 +5,7 @@ db_host = ${db_host}
 db_name = ${db_name}
 db_user = ${db_user}
 db_password = ${db_password}
+s3_useIam = ${use_iam_for_s3}
```

Comment:
s3_useIam = ${s3_useIam}

The variables inside the template are the variables injected into the template.